google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1beta2/stub/readrows/ReadRowsResumptionStrategy.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery.storage.v1beta2.stub.readrows;
+
+import com.google.api.core.InternalApi;
+import com.google.api.gax.retrying.StreamResumptionStrategy;
+import com.google.cloud.bigquery.storage.v1beta2.ReadRowsRequest;
+import com.google.cloud.bigquery.storage.v1beta2.ReadRowsResponse;
+import javax.annotation.Nonnull;
+
+/**
+ * An implementation of a {@link StreamResumptionStrategy} for the ReadRows API. This class tracks
+ * the offset of the last row received and, upon retry, attempts to resume the stream at the next
+ * offset.
+ *
+ * <p>This class is considered an internal implementation detail and not meant to be used by
+ * applications.
+ */
+@InternalApi
+public class ReadRowsResumptionStrategy
+    implements StreamResumptionStrategy<ReadRowsRequest, ReadRowsResponse> {
+
+  // Number of rows processed.
+  private long rowsProcessed = 0;
+
+  @Override
+  @Nonnull
+  public StreamResumptionStrategy<ReadRowsRequest, ReadRowsResponse> createNew() {
```

Comment:
is createNew idiomatic for our clients.  create seems shorter and conflate the two in my head.

Reply:
`createNew` is a method that is part of the StreamResumptionStrategy definition: http://googleapis.github.io/gax-java/1.30.0/apidocs/com/google/api/gax/retrying/StreamResumptionStrategy.html#createNew--

We are just overriding it and have no control over the naming.